generic/connectathon.py

```diff
@@ -51,12 +51,10 @@ def setUp(self):
         # Check for basic utilities
         smm = SoftwareManager()
         detected_distro = distro.detect()
-        packages = ['gcc', 'make']
+        packages = ['gcc', 'make', 'libtirpc-devel']
```

Comment:
Adding a devel package in the generic list is not a good idea. Please add it for appropriately for each distro.

Reply:
okay